pkgs/by-name/co/cortex-tools/package.nix

```diff
@@ -0,0 +1,41 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+buildGoModule (finalAttrs: {
+  pname = "cortex-tools";
+  version = "0.11.3";
+
+  src = fetchFromGitHub {
+    owner = "grafana";
+    repo = finalAttrs.pname;
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-+GWUC+lnCn5Nw2WytSvW/UsIMmMelCCsnKdBCHuue24=";
+  };
+
+  vendorHash = null;
+
+  subPackages = [
+    "cmd/benchtool"
+    "cmd/cortextool"
+    "cmd/e2ealerting"
+    "cmd/logtool"
+  ];
+
```

Comment:
Should add a `postInstall` to generate the man pages and completions for each tool.